### DISCUSSION TITLE
How to contribute to extend locales?

### DISCUSSION THREAD

**Discussion by devzom** [dargmuesli/nuxt-cookie-control#83] [Q&A] [answered]
I am up to take a action and add "PL" (Polish) translation?
How can I contribute, should contribute to the main author repo on gitlab?

**Comment by dargmuesli** [comment, marked as answer]
1. add a locale file like `src/runtime/locale/pl.ts`
2. import and export it in `src/runtime/locale/index.ts`
3. add it to the `Locale` type in `src/runtime/types.ts`
4. add it to the list of supported locales in the `README.md`

In general, please use [ISO 639-1 codes](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes).

**Reply by devzom** [reply]
I guess I was blind 😂 haven't seen that directory. Thanks!
